pyproject.toml

```diff
@@ -1,11 +1,110 @@
 [build-system]
-requires = ["setuptools<62", "pip", "wheel"]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=66,<68",
+    "setuptools_scm[toml]>=3.5.0",
+    "wheel",
+    "catalystcoop-pudl @ git+https://github.com/catalyst-cooperative/pudl@dev",
+    "fuzzywuzzy>=0.18,<0.19",
+    "python-levenshtein>=0.12,<0.13",
+    "recordlinkage>=0.14,<0.16",
+]
+
+[project]
+# This should be the *installed* package name e.g. catalystcoop.pudl not pudl
+name = "catalystcoop.pudl_rmi"
+description = "A one line description of the package."
+readme = {file = "README.rst", content-type = "text/x-rst"}
```

Comment:
Note that the README in this repository is currently written in markdown so it's `README.md` and will have a content type of `text/markdown` (I think).